service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -89,6 +90,11 @@
         super("cmo-progression");
     }
 
+    @BeforeEach
+    void setUp() {
+        futureDate = timeNow().plusDays(1);
```

Comment:
If you make this static you could move it to a `@BeforeClass` and then only need to do it once